### DISCUSSION TITLE
Unidirectional for Strimzi User Operator

### DISCUSSION THREAD

**Discussion by diogo-reis** [strimzi/strimzi-kafka-operator#10266] [Q&A] [answered]
Hi,

Is the UnidirectionalTopicOperator feature (or something similar) available for Strimzi User Operator?

I need to create and manage some users through the Strimzi User Operator while managing others directly through the Kafka CLI or via tools like Terraform (outside the scope of Strimzi).

However, when I do this, the users created directly in Kafka disappear whenever the User Operator's reconciliation runs. Is there a way to disable this behavior, allowing the Strimzi Operator to manage only the users it creates and not interfere with users created directly in Kafka?

I am using Strimzi version 0.39.0.

Thanks!

**Comment by scholzj** [comment, marked as answer]
No. The User Operator works like a traditional Kubernetes operator where the KafkaUser resources are the only source of truth. There are currently no plans to make it work like the Unidirectional Topic Operator.

**Reply by diogo-reis** [reply]
Thank you!
